src/library/components/stem-finder-result.js

```diff
@@ -53,16 +87,18 @@ var StemFinderResult = Component({
     });
 
     // mount/unmount lightbox outside of homepage content
-    if (lightbox) {
+    if (lightbox && page_scrolling == false) {
       var resourceLightbox = ResourceLightbox({
         resource: this.props.resource,
         toggleLightbox: this.toggleLightbox
       });
       Lightbox.open(resourceLightbox);
-      _gaq.push(['_trackEvent','Home Page Resource Card','Click',this.props.resource.name]);
+      //_gaq.push(['_trackEvent','Home Page Resource Card','Click',this.props.resource.name]);
```

Comment:
Did you mean disable this analytics event?

Reply:
No I didn't mean to leave it in.  I'll remove it.